workspaces/cli-server/src/analytics.ts

```diff
@@ -60,6 +61,12 @@ const inDevelopment = process.env.OPTIC_DEVELOPMENT === 'yes';
 // segment io sink
 const token = 'RvYGmY1bZqlbMukS8pP9DPEifG6CEBEs';
 const analytics = new Analytics(token);
+// Identify user
+getOrCreateAnonId().then((anonymousId) =>
```

Comment:
This should run on file load, which I think should work identically to the current implementation. I didn't port over the jwt decoding since we aren't using login, but if we reimplement a login, we can readd it here